app/views/profiles/_domain_user.erb

```diff
@@ -0,0 +1,57 @@
+<div class="w-full">
+  <div class="pb-4">
+    <h2 class="text-xl font-semibold leading-tight text-gray-900 dark:text-white">
```

Comment:
Just to give some more clarity, I reason I had it as an `h2` before was because that was supposed to be the title for the card (as email was the only thing in it my not-so-good label just titled it an email).  I would like to see this card still have a title, I will update the comment below with that.

Reply:
Fixed.